scripts/download-env.sh

```diff
@@ -383,6 +383,37 @@ check_version ${CPU_ARCH}-elf-gcc $GCC_VERSION
 #conda install -y $CONDA_FLAGS gdb-${CPU_ARCH}-elf=$GDB_VERSION
 #check_version ${CPU_ARCH}-elf-gdb $GDB_VERSION
 
+# Zephyr SDK
+################################################
+if [ "$FIRMWARE" = "zephyr" ]; then
+	VERSION="0.10.0"
```

Comment:
This version should go in https://github.com/timvideos/litex-buildenv/blob/master/scripts/settings.sh

Reply:
moved